### PR TITLE
fix ext-json for php 7.4

### PR DESCRIPTION
**Pull request by Axent96**
https://github.com/mediact/dependency-guard/issues/37

### CODE REVIEW

**Comment by willemstuursma** [comment]
Extensions should just be listed as `*` in `composer.json`. There is no need to have a specific version required.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
-    "name": "mediact/dependency-guard",
+    "name": "axent96/dependency-guard",
+    "replace": {
+        "mediact/dependency-guard":"1.1.0"
+    },
```
Comment (ksangers) [inline composer.json]: This shouldn't be part of the pull request

File: composer.json [hunk 2]
```diff
@@ -21,7 +24,7 @@
     "require": {
         "php": "^7.1",
         "ext-SPL": "^7.1",
-        "ext-json": "^1.5",
+        "ext-json": "^1.5 | ^7.4",
```
Comment (ksangers) [inline composer.json]: As @willemstuursma said, the version of `ext-*` should be `*`.

Comment (Axent96) [reply composer.json]: Yes, if the library work with ext-json < 1.5 then "*" is better. Must I make new pullRequest or you can change it without pullRequest?

File: composer.json (hunk 1 above)
Comment (Axent96) [reply composer.json]: sorry, I commited this after pull request. I must make the change but I wont use it temporary from fork version
